data-share/data-share-service/src/main/java/io/mosip/datashare/util/DigitalSignatureUtil.java

```diff
@@ -61,6 +61,10 @@ public class DigitalSignatureUtil {
 	@Value("${mosip.data.share.digest.algorithm:SHA256}")
 	private String digestAlg;
 
+	/** Defines whether JWT signature generation needs to be disabled */
+	@Value("${mosip.data.share.signature.disabled:false}")
+	private boolean isSignatureDisabled;

```

Comment:
Should we rather have this config in datashare service impl ? On a util class calling jwtSign to sign and giving back a empty string is not making sense.

Reply:
Done. Moved the logic for to the datashareImpl class.